internal/services/devtestlabs/dev_test_virtual_network_resource_test.go

```diff
@@ -203,6 +203,12 @@ resource "azurerm_dev_test_virtual_network" "test" {
   subnet {
     use_public_ip_address           = "Deny"
     use_in_virtual_machine_creation = "Allow"
+    shared_public_ip_address_configuration {
+      allowed_ports {
+        backend_port       = 22
+        transport_protocol = "Tcp"
+      }
```

Comment:
Multiple can be specified, can we please add another one to the test as well as test the update of these?

Reply:
Will add.